src/app.module.ts

```diff
@@ -2,10 +2,17 @@ import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
 import { PrismaModule } from './prisma/prisma.module';
+import { UserService } from './user/user.service';
+import { UserController } from './user/user.controller';
+import { AuthModule } from './auth/auth.module';
+import { UserModule } from './user/user.module';
+import { AuthController } from './auth/auth.controller';
+import { AuthService } from './auth/auth.service';
+import { LocalAuthGuard } from './auth/guards/local-auth.guard';
 
 @Module({
-  imports: [PrismaModule],
-  controllers: [AppController],
-  providers: [AppService],
+  imports: [PrismaModule, AuthModule, UserModule],
+  controllers: [AppController, UserController, AuthController],
+  providers: [AppService, UserService, AuthService, LocalAuthGuard],
```

Comment:
You should not need to add `UserController` or `AuthController` - `AppModule` is not using it. Same for `UserService` and `AuthService`. And I think `LocalAuthGuard` too but I'm not sure. They should be added to the feature modules they are used in, e.g. `UserController` should be added to the `UserModule` only. `AuthService` in `AuthModule` etc.